bitcoin/src/blockdata/transaction.rs

```diff
@@ -987,6 +987,51 @@ impl Transaction {
         // `Transaction` docs for full explanation).
         self.input.is_empty()
     }
+
+    /// Returns a reference to the input at `input_index` if it exists.
```

Comment:
A note about this being better than indexing by providing `Result` would be nice.

Reply:
Done.